comit/src/swap_id.rs

```diff
@@ -2,6 +2,55 @@ use serde::{Deserialize, Serialize};
 use std::{fmt, str::FromStr};
 use uuid::Uuid;
 
+/// This is an identifier created, and used locally, by a node to identify a
+/// swap to this node (as apposed to a ShareSwapId below).
+#[derive(Serialize, Deserialize, Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub struct LocalSwapId(Uuid);
```

Comment:
I'd be curious if we can model things in a way that don't require `LocalSwapId`.

Reply:
Just within the network module you mean, right?

Reply:
Yes.